setup.cfg

```diff
@@ -1,16 +1,11 @@
-[bdist_wheel]
-universal=1
-
-[tool:pytest]
-addopts = tests/
-
-[isort]
-include_trailing_comma = True
-multi_line_output = 3
-known_first_party = channels,daphne,asgiref,channels_redis
-line_length = 88
-
 [flake8]
 exclude = venv/*,tox/*,docs/*,testproject/*,js_client/*,.eggs/*
-ignore = E123,E128,E266,E402,W503,E731,W601
+extend-ignore = E123, E128, E266, E402, W503, E731, W601
 max-line-length = 120
+
+[isort]
+profile = black
+known_first_party = channels,daphne,asgiref,channels_redis,http_base,http_strategies
```

Comment:
We dropped `known_first_party` over in channels, if you're modifying the isort setup in this PR might as well copy that all at once.

Reply:
Sure, will do.